libs/designsystem/src/lib/components/card/card-header/card-header.component.ts

```diff
@@ -9,8 +9,15 @@ export class CardHeaderComponent implements OnInit {
   @Input() title: string;
   @Input() subtitle: string;
   @Input() isTitleBold: boolean;
+  @Input() flagged: 'success' | 'warning' | 'danger' | 'informational' = null;
 
   constructor() {}
 
   ngOnInit() {}
+
+  @HostBinding('class')
```

Comment:
Consider applying the `@HostBinding` on the `@Input() flagged` property instead (as it's a 1:1 mapping when removing the `flagged-` prefix)